target/linux/ar71xx/base-files/etc/board.d/01_leds

```diff
@@ -638,6 +638,12 @@ wnr612-v2)
 	ucidef_set_led_switch "lan1" "LAN1" "netgear:green:lan1" "switch0" "0x02" "0x0f"
 	ucidef_set_led_switch "lan2" "LAN2" "netgear:green:lan2" "switch0" "0x04" "0x0f"
 	;;
+wpa8630)
+	ucidef_set_led_default "power" "POWER" "$board:green:power" "1"
```

Comment:
Shouldn't be required. The power led is set to permanent on in diag.sh => set_state => done